src/Language/Haskell/Brittany/Internal.hs

```diff
@@ -505,10 +497,10 @@ ppModule lmod@(L _loc _m@(HsModule _name _exports _ decls _ _)) = do
     (ExactPrint.AnnComment (ExactPrint.Comment cmStr _ _), l) -> do
       ppmMoveToExactLoc l
       mTell $ Text.Builder.fromString cmStr
-    (ExactPrint.G AnnEofPos, (ExactPrint.DP (eofZ, eofX))) ->
+    (ExactPrint.G _, (ExactPrint.DP (eofZ, eofX))) ->
```

Comment:
I'm not sure if this change makes sense. `AnnEofPos` went away.

Reply:
Aha! `AnnEofPos` does still exist. It's no longer an `AnnKeywordId` defined by `ghc`. Rather it's a `KeywordId` defined by `ghc-exactprint`. I was confused by [the documentation](https://hackage.haskell.org/package/ghc-exactprint-0.6.4/docs/Language-Haskell-GHC-ExactPrint-Types.html#t:KeywordId), which doesn't show it as a constructor. However looking at [the source](https://hackage.haskell.org/package/ghc-exactprint-0.6.4/docs/src/Language.Haskell.GHC.ExactPrint.Types.html#KeywordId) reveals it behind CPP.